### PR TITLE
add pattern lab to web testing

### PR DESCRIPTION
**Pull request by vivianemartini**
## Adding a tooling library
- [ x] Suggestion is not a duplicate
- [x ] Suggestion is under the proper category (if applicable)
- [x ] Link to library is included 
- [x ] Name of library is included
- [x ] Name and link are spelled correctly

Happy Hacktoberfest 🎉

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -157,6 +157,9 @@ We would love to know which tooling library you think deserves to be included in
 - [React Testing Library](https://testing-library.com/)
   A popular testing library for React applications that also comes with a [Testing Playground](https://chrome.google.com/webstore/detail/testing-playground/hejbmebodbijjdhflfknehhcgaklhano).
 
+-[Pattern Lab](https://patternlab.io/)
```
Comment (jdwilkin4) [inline README.md:160]:
```suggestion
- [Pattern Lab](https://patternlab.io/)
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

**Comment by vivianemartini** [comment]
Hi @jdwilkin4, thank you for the feedback. I've implemented the changes ☺️

**Review by jdwilkin4 — APPROVED** [review]
Congrats on your first contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**